sentry.gradle

```diff
@@ -161,6 +193,20 @@ gradle.projectsEvaluated {
             bundleTask.finalizedBy cliTask
           }
           previousCliTask = cliTask
+          cliTask.finalizedBy modulesTask
+
+          def modulesCleanUpTask = tasks.create(name: nameModulesCleanup, type: Delete) {
+              description = "clean up collected modules generated file"
+              group = 'sentry.io'
+
+              delete modulesOutput
+          }
+
+          def packageTasks = tasks.findAll { task -> task.name.toLowerCase() == "package${variant}" && task.enabled }
```

Comment:
M: if `variant` isn't lowercase which I think the first letter is capitalized, this would fail.

Reply:
That's what I thought too, but when I checked the variant was `release` lowerCase.

I'll make it safer with `equalsIgnoreCase`.